### ISSUE TITLE
SVG patterns on scatterchart shapes

### ISSUE THREAD

**Issue by Serse** [novus/nvd3#1492]
I was wondering if it's possible to add SVG patterns on scatterchart shapes, like showed here http://stackoverflow.com/a/28255622

**Comment by liquidpele** [comment]
Yes, you can implement custom symbols to use with scatter.  In fact, the example even implements one to show... the 'X' is a custom symbol.  

https://nvd3-community.github.io/nvd3/examples/scatterChart.html

**Comment by Serse** [comment]
@liquidpele actually my question was on pattern fill: i'd like to fill circles with something like that https://iros.github.io/patternfills/sample_d3.html

**Comment by liquidpele** [comment]
Looks like it's just a CSS style, so sure, override the paths using CSS.

**Comment by Serse** [comment]
@liquidpele SVG doesn't support background images...

**Comment by liquidpele** [comment]
In the example you link to, it just sets a css fill.

**Comment by Serse** [comment]
Fill with a SVG pattern, as well described here http://stackoverflow.com/a/28255622
If you don't want to help me, please just leave the issue open. Thank you

**Comment by liquidpele** [comment]
Okay, if you feel like it's not answered then I'll reopen.
